validator/packaging/validator.toml.example

```diff
@@ -43,12 +43,12 @@ endpoint = "tcp://127.0.0.1:8800"
 
 # Uri(s) to connect to in order to initially connect to the validator network,
 # in the format tcp://hostname:port. This is not needed in static peering mode
-# and defaults to None.
-seeds = ["tcp://127.0.0.1:8801"]
+# and defaults to None. Replace host1 with the seed's hostname or IP address.
+seeds = ["tcp://host1:8800"]
```

Comment:
Should this be commented out, so as not to misconfigure a validator if copied into a running system?

Reply:
Sure--I can add that too.